apps/meteor/app/otr/client/OTRRoom.ts

```diff
@@ -110,6 +111,36 @@ export class OTRRoom implements IOTRRoom {
 		}
 	}
 
+	// Starts listening to other user's status changes and end OTR if any of the Users goes offline
+	// this should be called in 2 places: on acknowledge (meaning user accepted OTR) or on establish (meaning user initiated OTR)
+	listenToUserStatus(userId: IUser['_id']): void {
```

Comment:
Maybe we should use the `Presence` API on client side instead of directly listening to the stream.

For more context, please check the `usePresence` hook.